packages/commonwealth/client/scripts/models/MinimumProfile.tsx

```diff
@@ -1,5 +1,13 @@
 import jdenticon from 'jdenticon';
 
+export type UserProfile = {
+  id: number;
+  name: string;
+  address: string;
```

Comment:
Thanks for catching the issue with reactions. This is fixed now.

As to the comment above, the backend model returns a list of objects for now, but our frontend models map over each backend object to construct their own object. See for example in fetchThreads.ts, we have:
```
threads: res.data.result.threads.map((c) => new Thread(c)),
```

This is bad because now we have two source of truths, the model in our backend as well as our frontend. Ideally we should be using the backend data model directly like your suggestion. This is where trpc comes in. We can remove all these data holding classes like Thread and just use the response data directly in the client. But this will be a future cleanup effort.

Anyways if this didn't answer your question let me know!